clients/src/main/java/org/apache/kafka/clients/consumer/internals/Fetcher.java

```diff
@@ -193,15 +193,15 @@ public void updateFetchPositions(Set<TopicPartition> partitions) {
      * @param timeout time for which getting topic metadata is attempted
      * @return The map of topics with their partition information
      */
-    public Map<String, List<PartitionInfo>> getTopicMetadata(List<String> topics, long timeout) {
-        if (topics != null && topics.isEmpty())
+    public Map<String, List<PartitionInfo>> getTopicMetadata(List<String> topics, boolean allTopics, long timeout) {
```

Comment:
Since this was in internals and I needed to update all usages regardless, I located "related" parameters close to each other. I can move to the  end if you prefer.